src/webapp/features/home/index.tsx

```diff
@@ -18,14 +19,17 @@ const Home = () => {
   }
 
   return (
+    <React.Fragment>
```

Comment:
Ooh here's something fun - here's an even better syntax for React.fragment:
```
    <>
      <RabbleLogo />
      ...
      </div>
    </>
```

Let's change it to use this less verbose syntax!!  🏄